### DISCUSSION TITLE
Marketing analytics icons

### DISCUSSION THREAD

**Discussion by darcimm** [twilio-labs/paste#3355] [Request: Icon] [answered]
### Description

Hi there! We have various 'sent' 'queued' and 'delivered' icons for channels like email, SMS, and Push notifications.

Here's an example for push notifications:
<img width="533" alt="image" src="https://github.com/twilio-labs/paste/assets/83974127/7b4b3bdc-f06c-42d5-9edb-d74014f4e1ae">

Here's an example of how the icons are used:
<img width="709" alt="image" src="https://github.com/twilio-labs/paste/assets/83974127/c1c2bb40-f780-46d7-bb4d-314eebc6bcc9">

Wanted to understand whether we should go ahead and self-serve or if something like this needs to be added to Paste? 

Thanks!
Darci

### Link to icon

https://www.streamlinehq.com/icons/legacy-streamline-regular?search=bell

### Icon type

- [X] UI icon
- [ ] Product icon

**Comment by loreina** [comment]
Hi @darcimm,

Could you share additional screenshots or a Figma file so we can see the icons in more context? Thanks!

**Reply by darcimm** [reply]
Here's a link to [Figma](https://www.figma.com/file/npHoqZDj3hLxMQ8dbY1eaH/%5BETP%5D-Marketing-Analytics?type=design&node-id=3399-397984&mode=design&t=fAHXYrG3QDfQLHEp-4)!

**Comment by loreina** [comment]
Hi @darcimm,

I spoke with the team and we've added it to our [backlog](https://twilio-productivity.atlassian.net/browse/DS-191) to add generic icons for sent, queued, and delivered. Could you file a [dependency request](https://airtable.com/shrvMvL3BG6Zm0f9l) for the icons as well? That's our official intake form to keep track of dependencies and prioritize requests based on needed-by dates.

The team will follow up once the icons are in Figma and code. Cheers!

**Comment by serifluous** [comment, marked as answer]
SentIcon and DeliveredIcon are now available in Paste! @darcimm @joelbandi

In Figma, you'll have to wait a week or so to be able use them because we have some publishing issues in the Paste Components library that the Figma team is working on fixing right now.

**Reply by serifluous** [reply]
The icons have been published in Figma!
